enclave_build/src/image_manager.rs

```diff
@@ -6,8 +6,25 @@ use crate::image::ImageDetails;
 use crate::storage::OciStorage;
 use crate::{EnclaveBuildError, Result};
 
+use tempfile::NamedTempFile;
```

Comment:
Can we use the one from vmm-sys-util?

Reply:
For now we match the `load` operation from the Docker workflow. I suggest we change it everywhere in another PR and remove the `tempfile` dependency altogether. Right now I would have to modify too many modules that are not related to these changes.

Reply:
Got it, thanks